test/config/sequencer.js

```diff
@@ -0,0 +1,18 @@
+const Sequencer = require('@jest/test-sequencer').default
+
+class TestSequencer extends Sequencer {
+  sort(tests) {
```

Comment:
All tests in the `dapp-library.spec.ts` require an existing account. So to avoid creating another account and funding it, it can be executed at the end and use the one created previously in other test.

Reply:
just create another account and fund it in the beforeAll hook of the `dapp-library.spec.ts`. Tests shouldn't be dependent on each other.

Reply:
There is just one problem. We are currently installing fdp-storage from github, because of the bee issue. But there is only browser version of the fdp-storage. When it's imported inside of a test, it throws an error, because it's not suitable for node apps. So writing a script to create an account won't work currently. And creating account by repeating all the steps from other tests would be too much unneeded work.